### PR TITLE
Making library compatible with actix-web 1.0

### PR DESCRIPTION
**Pull request by hstefan**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -21,10 +21,10 @@ path = "src/lib.rs"
 [dependencies]
 bytes = "0.4"
 futures = "0.1"
-failure = "0.1"
+derive_more = "0.14"
 
-actix = "0.7"
-actix-web = "0.7"
+actix = "0.8.1"
+actix-web = "1.0.0-b3"
```
Comment (fafhrd91) [inline Cargo.toml]: 1.0.0-beta5

**Review by fafhrd91 — CHANGES_REQUESTED** [review]
could you also remove travis config for 1.21

**Comment by hstefan** [comment]
@fafhrd91 should be good now!

**Comment by fafhrd91** [comment]
looks good. could you add entry to changes

**Review by fafhrd91 — APPROVED** [review]

**Comment by hstefan** [comment]
@fafhrd91 done, should be good to merge now

File: Cargo.toml (hunk 1 above)
Comment (otavio) [reply Cargo.toml]: `1.0.0-rc` now

Comment (hstefan) [reply Cargo.toml]: Thinks move so fast! Bumped again.

**Review by fafhrd91 — APPROVED** [review]

**Comment by fafhrd91** [comment]
thanks!

**Pull request merged**